azureeyemodule/app/model/objectdetector.hpp

```diff
@@ -28,6 +28,7 @@ class ObjectDetector : public AzureEyeModel
 public:
     ObjectDetector(const std::string &labelfpath, const std::vector<std::string> &modelfpaths, const std::string &mvcmd, const std::string &videofile, const cv::gapi::mx::Camera::Mode &resolution);
     virtual ~ObjectDetector();
+    const std::string VIDEO_PREFIX = "video:";
```

Comment:
Doesn't the user pass in the "video:" as part of the arg on the command line?

Reply:
Yes. so I am using this prefix to parse the file path behind this keyword. Does this answer your question?

Reply:
I don't know. I just figured if the user was already writing this, why do we need it here. But also, if it works, I'm not going to worry about it.